conode/run_conode.sh

```diff
@@ -26,6 +26,12 @@ VERSION="$VERSION_ONET-$VERSION_SUB"
 #   TAGS="-tags vartime" ./run_conode.sh
 # Note: TAGS is also used by the integration tests.
 
+# This will allow you to put the servers on a different range if you want.
+#   $ export PORTBASE=18000
+#   $ ./run_conode.sh local 3
+# Results in the servers being on 18002-18007.
+[ -z "$PORTBASE" ] && PORTBASE=7000
+
```

Comment:
We absolutely need to kill `run_conode.sh` - it really gets aweful. Of course it's OK to extend it like you do, no need to change - but `run_conode.sh` should be removed...